CMakeLists.txt

```diff
@@ -8,9 +8,18 @@ cmake_minimum_required(VERSION 3.1) # for "CMAKE_CXX_STANDARD" version
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake") # custom CMake modules like FindSQLiteCpp
 project(SQLiteCpp VERSION 2.99)
 
-# SQLiteC++ 3.x now requires C++11 compiler
-set(CMAKE_CXX_STANDARD 11)
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
+# SQLiteC++ 3.x can make use of C++11 features
+if (NOT CMAKE_CXX_STANDARD)
+    set(CMAKE_CXX_STANDARD 11)
+endif ()
+# googletest requires C++11 features
```

Comment:
SQLiteCpp 3.x on master branch already requires C++11

Reply:
Ok, then I will make `CMAKE_CXX_STANDARD_REQUIRED ON` unconditional, and I can add a warning if users set a lower standard than `11`.